src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
     }
```

Comment:
Creating a new instance of `Random` every time the method is called is inefficient and unnecessary. You should create a single instance of `Random` as a class-level field and reuse it for generating random numbers.